lib/build/raty.js

```diff
@@ -667,7 +666,7 @@ var Raty = function () {
   }, {
     key: "_getWidth",
     value: function _getWidth() {
-      var width = this.stars[0].width || parseFloat(this.stars[0].offsetWidth);
+      var width = parseFloat(this.stars[0].offsetWidth);
```

Comment:
Why did we remove it?

Reply:
@wbotelhos it remove because this.stars[0].offsetWidth do the both things here which is done by this.stars[0].width and this.stars[0].fontSize. I just reduce the line to a single line but the behaviour is the same.